docs/docs/general/07-Troubleshooting/windows.md

```diff
@@ -0,0 +1,15 @@
+# Windows
+
+## askui Controller 
+
+On windows the problem occurs that the askui Controller doesn't terminate after test execution.
```

Comment:
```suggestion
On Windows, the askui UI Controller may not terminate after test execution. This may lead to problems when using the askui UI Controller in a pipeline like a pipeline's run not coming to an end. There are 2 ways to deal with this issue:

- configuring Jest to force the exit of child processes, or,
- ...
```